website/blog/2022-09-15-announcement-v2.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: announcement-v2
+title: Introducing Remirror v2 🎉
+author: Ronny Roeller
+author_title: Remirror Maintainer
+author_url: https://github.com/ronnyroeller
+author_image_url: https://avatars.githubusercontent.com/u/9339055?v=4
+tags: [remirror, announcement]
+---
+
+_TLDR: Remirror adopts Prosemirror TypeScript support._
+
+<!-- truncate -->
+
+Introducing Remirror v2 🎉
+
+### 🧐 Why?
+
+Remirror stands on the shoulders of giants, most notably Prosemirror itself. We therefore want our community to timely benefit whenever one of these great projects adds new features, bug fixes, and the like.
+
+Recently, Prosemirror released [native Typescript support](https://discuss.prosemirror.net/t/prosemirror-is-now-a-typescript-project/4624). In itself this is awesome news. Unfortunately the types differ from the ones from DefinitelyTyped, which underlay many of Remirror's types. This means that we must update our types to not get stuck on an old Prosemirror version.
+
+The bottom line is: Prosemirror basically forced our hand to release a new major version rather soon. Be that as it may, we want to make the most out of this situation! It allows us to move forward some other breaking changes that help the project.
+
+### ✨ What's new?
+
+First, we **removed deprecated features** for good:
+
+- useEvents (plural)
```

Comment:
Missing `useEvent` (singular)

Use `useEditorEvent` instead, this prevents a name clash with the React RFC of the same name.